virt-v2v/cold/entrypoint.go

```diff
@@ -0,0 +1,207 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+const (
+	OVA     = "ova"
+	vSphere = "vSphere"
+	DIR     = "/var/tmp/v2v"
+	FS      = "/mnt/disks/disk[0-9]*"
+	Block   = "/dev/block[0-9]*"
+	VDDK    = "/opt/vmware-vix-disklib-distrib"
+)
+
+var (
+	xmlFilePath string
+	server      *http.Server
+)
+
+func main() {
+	virtV2vArgs := []string{"virt-v2v", "-v", "-x"}
+	source := os.Getenv("V2V_source")
+
+	requiredEnvVars := map[string][]string{
+		vSphere: {"V2V_libvirtURL", "V2V_secretKey", "V2V_vmName"},
+		OVA:     {"V2V_diskPath", "V2V_vmName"},
+	}
+
+	if envVars, ok := requiredEnvVars[source]; ok {
+		if !checkEnvVariablesSet(envVars...) {
+			fmt.Printf("Following environment variables need to be defined: %v\n", envVars)
+			os.Exit(1)
+		}
+	}
+
+	switch source {
+	case vSphere:
+		virtV2vArgs = append(virtV2vArgs, "-i", "libvirt", "-ic", os.Getenv("V2V_libvirtURL"))
+		pwFilePath := fmt.Sprintf("%s/vmware.pw", DIR)
```

Comment:
the entrypoint script [created the DIR folder](https://github.com/kubev2v/forklift/blob/main/virt-v2v/cold/entrypoint#L44), don't we need to do that here?